docs/data/base/components/button/button.md

```diff
@@ -34,6 +34,21 @@ In addition to HTML elements, you can also use SVGs with the `ButtonUnstyled` co
 
 {{"demo": "UnstyledButtonCustom.js"}}
 
+## Focus of disabled buttons
+
+Similarly to the native `<button>`, the `ButtonUnstyled` component can't receive focus when it's disabled.
```

Comment:
Maybe expand on when this would be a good use-cases (for example `MenuItem`) and why. For example, in some cases, disabled elements that are required for completing a step (or are just generally too important to be missed) should be focusable with TAB key.

Reply:
:+1: I explained how this may be important for accessibility and shared that we use this internally for the menu button. Does it look better for you now?